modules/apps/document-library/document-library-service/src/main/java/com/liferay/document/library/internal/repository/capabilities/LiferayVersioningCapability.java

```diff
@@ -223,9 +222,6 @@ private FileEntry _purgeVersions(
 		return fileEntry;
 	}
 
-	@Reference
-	private DLConfiguration _dlConfiguration;
```

Comment:
@JanisZhang This commit did remove the usage to the field, but added `@Reference` annotation to it. This is strange as we usually don't declare dependency on a configuration this way. Instead, we use `configurationPid` property in the `@Component` annotation.

I think this field can be removed, as it has a 1..1 reference to `VersionPurger` which depends on the config. But please modify your commit message to be accurate.

Reply:
@dantewang Got it. THX